ui/app/templates/jobs/run/templates/new.hbs

```diff
@@ -0,0 +1,34 @@
+<section class="section">
+    <form class="new-job-template" autocomplete="off">
+        <div class={{if this.namespaceOptions "path-namespace"}}>
+            <label>
+            <span>
+                Template name
+            </span>
+            <Input
+                @type="text"
+                @value={{mut this.templateName}}
+                placeholder="your-template-name-here"
+                class="input path-input"
+                {{autofocus}}
+                data-test-template-name
+            />
+            </label>
+            <SingleSelectDropdown
```

Comment:
Running into a fatal error here: 
![image](https://user-images.githubusercontent.com/713991/211635723-9982e50a-ce75-498d-b739-593995c0ff15.png)

(I don't have namespaces other than default)

I think you probably want to wrap this in a `{{#if this.system.shouldShowNamespaces}}` conditional

Reply:
Should be patched by some conditional logic here for when `namespaces` are enabled.